### ISSUE TITLE
Start Page: Ben & Yonghyun

### ISSUE THREAD

**Issue by benriexinger** [jacondel/memory1#1]
Creating an interface with Google API to use logins.  Also, create a client side page to host the interaction.

**Comment by yonhyun** [comment]
<!DOCTYPE html>
<html lang="en">
<head>
<title>Bootstrap Theme Company Page</title>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
  <link href="http://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet" type="text/css">
  <link href="http://fonts.googleapis.com/css?family=Lato" rel="stylesheet" type="text/css">
  <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css" rel="stylesheet" />
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
  <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
  <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
  <style>

  	.jumbotron {
      background-color: #045FB4;
      color: #fff;
      padding: 140px 25px;
      font-family: Montserrat, sans-serif;
  }
    .col-centered {
    display:inline-block;
    float:none;
    /* reset the text-align */
    text-align:left;
    /* inline-block space fix */
    margin-right:-4px;
}
.col-centered{
    float: none;
    margin: 0 auto;
}

  </style>

  </head>
  <body>

  <script src="https://apis.google.com/_/scs/apps-static/_/js/k=oz.gapi.en.kktq_lQT_ok.O/m=signin2/rt=j/sv=1/d=1/ed=1/am=AQ/rs=AGLTcCN3KeQOiCZFRaPdw9QOhcptK5_rsg/cb=gapi.loaded_0" async=""></script>
  <script src="https://apis.google.com/js/platform.js" async="" defer="" gapi_processed="true"></script>
  <meta name="google-signin-client_id" content="316690487581-0h7pupate7blhm3uobof64b9cl5sfns5.apps.googleusercontent.com">


  <div class="jumbotron text-center">
  <h1>Memory Project</h1>
  <p>Group project for CSE421</p>
  <form class="form-inline" name=test>
    <input type="text" id="test" class="form-control" size="50" placeholder="Your name" required>
    <p></p>


  
   



    
  </form>
   <p></p>

   <div class="row">
  <div class="col-md-1"></div>
  <div class="col-md-1"></div>
  <div class="col-md-1">.</div>
  <div class="col-md-1"></div>
  <div class="col-md-1"></div>
  <div class="col-md-1"></div>
  <div class="col-md-1"><div class="g-signin2" data-onsuccess="onSignIn"></div></div>
  <div class="col-md-1"></div>
  <div class="col-md-1"></div>
  <div class="col-md-1"></div>
  <div class="col-md-1"></div>
  <div class="col-md-1"></div>
</div>



   


       

</div>




<script>

	function myfunc(){
		var user=document.getElementById("test");
		var hi='Fuck me so bad, ';
		alert(hi+user.value);
    location.href="http://xonline.tv";
	}

</script>
<script>
  
  function onSignIn(googleUser) {
                var profile = googleUser.getBasicProfile();
                document.getElementById("d1").innerHTML="Welcome: " + profile.getName();
                console.log('ID: ' + profile.getId()); // Do not send to your backend! Use an ID token instead.
                console.log('Name: ' + profile.getName());
                console.log('Image URL: ' + profile.getImageUrl());
                console.log('Email: ' + profile.getEmail());
                window.sessionStorage.setItem("personName", profile.getName());
                window.sessionStorage.setItem("personEMail", profile.getEmail());
                window.location = "loggedIn.html";
        }

</script>






  </body>
  </html>
